test/data-source.test.js

```diff
@@ -103,7 +102,9 @@ describe('DataSource', function() {
         var fn = scope[name];
         var actuallyEnabled = Model.getRemoteMethod(name);
         assert(fn, name + ' should be defined!');
-        assert(actuallyEnabled === isRemoteEnabled, name + ' ' + (isRemoteEnabled ? 'should' : 'should not') + ' be remote enabled');
+        assert(actuallyEnabled === isRemoteEnabled,
+               name + ' ' + (isRemoteEnabled ? 'should' : 'should not') +
+                 ' be remote enabled');
```

Comment:
We should just not indent (like you did above). Maybe to keep the rule simple, don't indent when you wrap long lines.

However, most LoopBack code I see indents one time when wrapping long lines (I believe @raymondfeng  does it this way), but we end up indenting too much this way IMO.

I generally follow the rules from the Google JS style guide (https://google.github.io/styleguide/javascriptguide.xml - search "indenting wrapped lines").

```
// your new suggested style
function(a, b,
c, d) {
  hello("some code with super long lines +
  "and more super long lines");
    helloAgain("some more code with super long lines
    "and more super long lines");
  

//current LB

function(a, b
  c, d) {
    hello("some code with super long lines +
      "and more super long lines");
        helloAgain("some code with super long lines +
          "and more super long lines");

//google style guide
function(a, b,
    c, d) {
  hello("some code with super long lines +
      "and some more super long lines");
    helloAgain("some code with super long lines +
        "and some more super long lines");

```